app/models/miq_worker.rb

```diff
@@ -358,16 +367,35 @@ def containerized_worker?
     self.class.containerized_worker?
   end
 
+  def self.supports_systemd?
+    worker_settings[:systemd_enabled]
+  end
+
+  def self.systemd_worker?
```

Comment:
Since you extracted this to a helper method can you do the same for the container stuff ? like here - https://github.com/ManageIQ/manageiq/pull/18648/files#diff-79963741e7c656b8ddb47a9e91a1af6eR39

This way it keeps symmetry in that method (I almost commented about that above before stumbling on it here)

Reply:
Actually it was already extracted to a `containerized_worker?` method, just need to actually use it over there in the quiesce code

Reply:
Done